### ISSUE TITLE
OSV-2021-557.yaml has incorrect fixed attribute

### ISSUE THREAD

**Issue by ddkilzer** [google/oss-fuzz-vulns#30]
[OSV-2021-557.yaml](https://github.com/google/oss-fuzz-vulns/blob/main/vulns/libarchive/OSV-2021-557.yaml) has incorrect fixed attribute.

TL;DR:
- Change fixed commit to: `17f4e83c0f0fc3bacf4b2bbacb01f987bb5aff5f`
- Remove `3.5.3` from the list of affected versions as it contains the fix.

Details:

Currently the fixed attribute claims this was resolved by this commit, which has nothing to do with libarchive:  [56c920eab3352f7877ee0cf9e472c1ab376c7e3e](https://github.com/libarchive/libarchive/commit/56c920eab3352f7877ee0cf9e472c1ab376c7e3e)

Based on some research, the actual fix was made in <https://github.com/libarchive/libarchive/pull/1491> which landed in:  [17f4e83c0f0fc3bacf4b2bbacb01f987bb5aff5f](https://github.com/libarchive/libarchive/commit/17f4e83c0f0fc3bacf4b2bbacb01f987bb5aff5f)

However, this fix was also cherry-picked to v3.5.3 in:  [05591dd516aa454e6d37fc55a2facac5f91355eb](https://github.com/libarchive/libarchive/commit/05591dd516aa454e6d37fc55a2facac5f91355eb)

So v3.5.3 should also be removed from affected versions.

Finally, note that <https://github.com/libarchive/libarchive/pull/1492> and <https://github.com/libarchive/libarchive/pull/1493> are both closely related, and they were both cherry-picked into v3.5.3.

**Comment by oliverchang** [comment]
Thanks! 

This is now updated in https://github.com/google/oss-fuzz-vulns/blob/main/vulns/libarchive/OSV-2021-557.yaml
